azure/KeyVaultUploader.cs

```diff
@@ -9,21 +9,21 @@ internal class KeyVaultUploader : ISecretUploader
     {
         private readonly ILogger _logger;
         private readonly SecretClient _secretClient;
-        private readonly KeyOptions _keyOptions;
+        private readonly KeyVaultOptions _options;
 
         public KeyVaultUploader(
-            KeyOptions options,
+            KeyVaultOptions options,
             TokenCredential credential,
             ILogger<KeyVaultUploader> logger)
         {
-            _keyOptions = options;
+            _options = options;
             _logger = logger;
             _secretClient = new SecretClient(options.KeyVaultUri, credential);
```

Comment:
From the code in the caller side, seems like KeyVaultUploader instance is always created now even if the KeyVaultUri is not set. Just curious, does SecretClient work well if the KeyVaultUri is empty or null?

Reply:
No. the SecreteClient will throw an exception. Fixed this and pushed an update.